Help/new.html.subtemplate

```diff
@@ -64,10 +64,10 @@
     <p>
       <strong>
         <span class="glyphicon glyphicon-eye-open"></span> 
-        Experts Only:
+        Indie Hosts Only:
       </strong> 
       Other applications built on Privly provide privacy only in special 
-      circumstances and should be viewed as "experts only."
+      circumstances and should be viewed as "Indie Hosts Only."
```

Comment:
How about for the description: "These applications provide privacy if you host your own content server. If you don't know what that means, you should not use these apps."